pyigm/utils.py

```diff
@@ -382,3 +382,32 @@ def velcorr_mould(galaxy,cosmo=None):
                        distmod=distmod, flag=flag, scale=scale.to(u.kpc/u.arcsec))
 
     return velcorrdict
+
+def get_components(obj,ion,zrange=None):
+    '''Return list of components from AbsSystem or IGMSightline for given ion
```

Comment:
I believe the convention is to use """ instead of '''.

Reply:
Changed.  Although I've probably used ''' in other places.  Will try and watch out for that.